src/GraphQL/ResolveFieldContext/IResolveFieldContext.cs

```diff
@@ -39,6 +39,14 @@ public interface IResolveFieldContext : IProvideUserContext
         /// </summary>
         IDictionary<string, ArgumentValue>? Arguments { get; }
 
+        /// <summary>
+        /// A dictionary of directives with their arguments passed to the field, or <see langword="null"/> if no directives were defined for the field.
+        /// It is recommended to use the <see cref="ResolveFieldContextExtensions.GetDirective(IResolveFieldContext, string)">GetDirective</see>
+        /// and <see cref="ResolveFieldContextExtensions.HasDirective(IResolveFieldContext, string)">HasDirective</see> extension
+        /// methods rather than this dictionary directly.
```

Comment:
" so the names can be converted by the selected <see cref="INameConverter"/" was removed